src/manifolds/VectorBundle.jl

```diff
@@ -274,7 +274,14 @@ function exp!(B::VectorBundle, q, p, X)
     VXM, VXF = submanifold_components(B.manifold, X)
     # this temporary avoids overwriting `p` when `q` and `p` occupy the same memory
     xqt = exp(B.manifold, xp, VXM)
```

Comment:
You can do this one I place again, since `xq` is not used below anymore.

Reply:
I also thought so at first but actually I can't because `xp` (which may refer to the same memory) is used below.

Reply:
Ah, right, I again forgot that they share memory eventually. Then this has to stay as it is.